src/pages/components/tabs/usage.mdx

```diff
@@ -323,23 +336,32 @@ tab to the edge of the content area.
 
 ### States
 
-Tabs allow for three states: **selected, unselected,** and **disabled**. The
-default view is that one tab is preselected and is usually the first tab. Only
-one tab can be selected at a time. When a user chooses a new item, the previous
-tab is automatically deselected. If a user navigates away from a tab, a user
-should return to the last tab selected.
+Tabs have two states—**selected,** and **unselected,**. Other interactive states
+are **hover, focus,** and **disabled**. The default view is one tab is
+preselected and is usually the first tab. Only one tab can be selected at a
+time. When a user chooses a new item, the previous tab is automatically
+deselected. If a user navigates away from a tab, a user should return to the
+last tab selected.
 
 For detailed visual information about the various states for this component, see
 the [Style tab](https://carbondesignsystem.com/components/tabs/style).
 
+| State      | When to use                                                                                                                                                                                                                                              |
+| ---------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Selected   | When a user clicks or uses the arrows keys to activate tab.                                                                                                                                                                                              |
+| Hover      | When a user is hovering over the tag with the mouse cursor to interact with it.                                                                                                                                                                          |
+| Focus      | When a user clicks on the tag or navigates using the keyboard with left and right arrows, it becomes focused, indicating the user has successfully navigated to the component.                                                                           |
+| Unselected | When the user navigates to another tab and the tab is inactive.                                                                                                                                                                                          |
+| Disabled   | When a user is not allowed to interact with the tab due to either permissions, dependencies, or pre-requisites. The disabled state completely removes the interactive function from a component. The styling is not subject to WCAG contrast compliance. |
+
 <Row>
 <Column colLg={8}>
 
-![Tab states](images/tab-usage-10.png)
+![Example of line and contained tab selected, unselected, and disabled states](images/tab-usage-10.png)
```

Comment:
I think we could remove this image if you want. It also doesn't currently show the hover and focus states that are listed in the table above. We have started to do this on some of the Usage tabs when we provide a link out to the Style tab for people to reference the more in-depth state images. I think we are starting to especially do this when there are many variants of one component.